src/test/java/com/amazon/sqs/javamessaging/AmazonSQSExtendedClientTest.java

```diff
@@ -441,79 +458,103 @@ public void testExtendedClientConfiguredDoesNotDeleteObjectFromS3UponDelete() {
         String randomS3Key = UUID.randomUUID().toString();
         String originalReceiptHandle = UUID.randomUUID().toString();
         String largeMessageReceiptHandle = getLargeReceiptHandle(randomS3Key, originalReceiptHandle);
-        DeleteMessageRequest deleteRequest = new DeleteMessageRequest(SQS_QUEUE_URL, largeMessageReceiptHandle);
+        DeleteMessageRequest deleteRequest = DeleteMessageRequest.builder().queueUrl(SQS_QUEUE_URL).receiptHandle(largeMessageReceiptHandle).build();
 
         ExtendedClientConfiguration extendedClientConfiguration = new ExtendedClientConfiguration()
                 .withPayloadSupportEnabled(mockS3, S3_BUCKET_NAME, false);
 
-        AmazonSQS extendedSqs = spy(new AmazonSQSExtendedClient(mockSqsBackend, extendedClientConfiguration));
+        SqsClient extendedSqs = spy(new AmazonSQSExtendedClient(mockSqsBackend, extendedClientConfiguration));
 
         // when
         extendedSqs.deleteMessage(deleteRequest);
 
         // then
         ArgumentCaptor<DeleteMessageRequest> deleteRequestCaptor = ArgumentCaptor.forClass(DeleteMessageRequest.class);
         verify(mockSqsBackend).deleteMessage(deleteRequestCaptor.capture());
-        Assert.assertEquals(originalReceiptHandle, deleteRequestCaptor.getValue().getReceiptHandle());
+        Assert.assertEquals(originalReceiptHandle, deleteRequestCaptor.getValue().receiptHandle());
         verifyZeroInteractions(mockS3);
     }
 
     @Test
     public void testExtendedClientConfiguredDoesNotDeletesObjectsFromS3UponDeleteBatch() {
         // given
         int batchSize = 10;
-        DeleteMessageBatchRequest deleteBatchRequest = generateLargeDeleteBatchRequest(batchSize);
+        List<String> originalReceiptHandles = IntStream.range(0, batchSize)
+            .mapToObj(i -> UUID.randomUUID().toString())
+            .collect(Collectors.toList());
+        DeleteMessageBatchRequest deleteBatchRequest = generateLargeDeleteBatchRequest(originalReceiptHandles);
         ExtendedClientConfiguration extendedClientConfiguration = new ExtendedClientConfiguration()
                 .withPayloadSupportEnabled(mockS3, S3_BUCKET_NAME, false);
-        AmazonSQS extendedSqs = spy(new AmazonSQSExtendedClient(mockSqsBackend, extendedClientConfiguration));
+        SqsClient extendedSqs = spy(new AmazonSQSExtendedClient(mockSqsBackend, extendedClientConfiguration));
 
         // when
         extendedSqs.deleteMessageBatch(deleteBatchRequest);
 
         // then
-        verify(mockSqsBackend, times(1)).deleteMessageBatch(any(DeleteMessageBatchRequest.class));
+        ArgumentCaptor<DeleteMessageBatchRequest> deleteBatchRequestCaptor = ArgumentCaptor.forClass(DeleteMessageBatchRequest.class);
+        verify(mockSqsBackend, times(1)).deleteMessageBatch(deleteBatchRequestCaptor.capture());
+        DeleteMessageBatchRequest request = deleteBatchRequestCaptor.getValue();
+        Assert.assertEquals(originalReceiptHandles.size(), request.entries().size());
```

Comment:
i improved the test to check that the receipt handles were getting passed along correctly